x/staking/alias.go

```diff
@@ -18,11 +18,14 @@ type (
 	Commission              = types.Commission
 	CommissionMsg           = types.CommissionMsg
 	Delegation              = types.Delegation
+	DelegationResp          = types.DelegationResp
```

Comment:
Why do we somewhere use the `Resp` suffix and elsewhere `Responses` :-/ ?

Reply:
I meant: e.g. why don't we just use here `DelegationResponse`?

Reply:
It doesn't have to be fixed in this PR obviously

Reply:
We can @alessio -- I'm indifferent.

Reply:
Would you like me to make said change?

Reply:
That'd be great